src/patternfly/upgrade-examples/LoginUpgradeExamples/examples/login-upgrade-examples-pf3-login.hbs

```diff
@@ -0,0 +1,58 @@
+{{#> login-upgrade-examples}}
+
```

Comment:
You're missing `.login-pf` and `.login-pf-page` in the code. It probably makes sense for this to be a full-page example, since it's a full page component and the pf4 login component is, too.

Reply:
Additional class names has been added to the table of class usage descriptions. Also, I agree with you on outputting PF3 login example as an external linked. But as requested by @dgutride, we're putting the correct linkage on hold until the new PF4 site is up.

Reply:
Oh I mean you're missing them in the markup, too. And re: an external link, I just mean adding `fullPageOnly="true"` to the example component so this links in a new window, like the PF4 login demo.

Reply:
Although I just read @christiemolloy's comment about including the code in the upgrade page. I agree that it would be nice to have the code on the upgrade page to reference, but since they're full-page components, I'm not sure if it makes sense to have them on the same page?

Reply:
This has definitely been a topic of discussion in the past with @dgutride, @christiemolloy, @matthewcarleton. But from what I remember, we're just getting this out the door so that we have pieces to work on when we finally decide on how the login works, looks, and feels on the new PF4 site.